layers/utils/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/Utils.java

```diff
@@ -141,6 +141,17 @@ public static String toJson(Object obj) {
         return json;
     }
 
+    public static boolean isCnRegion(final String region) {
+        return region.startsWith("cn-");
+    }
+
+    public static String getUrlSuffix(final String region) {
+        if (isCnRegion(region)) {
+            return "amazonaws.com.cn";
+        }
+        return "amazonaws.com";
+    }
```

Comment:
Why not use the `isChinaRegion(String)` and `endpointSuffix(String)` functions already in this file?

Reply:
change the code to use isChinaRegion(String) and endpointSuffix(String) functions